Model/Models/Validators/PlanValidator.cs

```diff
@@ -5,45 +5,63 @@ namespace Model.Models.Validators
 {
     public class PlanValidator
     {
+        #region Validators
         public static bool ValidateCreatePlan(Plan plan)
         {
-            if (plan.PlanId != null)
+            if (PlanExists(plan))
             {
                 return false;
             }
-            if (plan.CoachId == null)
+            if (!CoachExists(plan))
             {
                 return false;
             }
             return true;
         }
         public static bool ValidateGetPlan(string planId)
         {
-            if (planId == null)
+            if (!PlanExists(planId))
             {
                 return false;
             }
             return true;
         }
         public static bool ValidateUpdatePlan(Plan plan)
         {
-            if (plan.PlanId == null)
+            if (!PlanExists(plan))
             {
                 return false;
             }
-            if (plan.CoachId == null)
+            if (!CoachExists(plan))
             {
                 return false;
             }
             return true;
         }
         public static bool ValidateGetWorkouts(string planId)
         {
-            if (planId == null)
+            if (!PlanExists(planId))
             {
                 return false;
             }
             return true;
         }
+        #endregion
+
+        #region Helper Functions
+        private static bool PlanExists(Plan plan)
+        {
+            return plan.PlanId != null;
```

Comment:
can you change this to 
`return (plan.PlanId != null) && (plan.planId != "");`
I think I forgot to do this on the CommentValidator but we should be checking for empty string IDs